README.md

```diff
@@ -277,41 +274,47 @@ keywords = [
   "token",
 ]
 
-# You can include an allowlist table for a single rule to reduce false positives or ignore commits
-# with known/rotated secrets
-[rules.allowlist]
-description = "ignore commit A"
-commits = [ "commit-A", "commit-B"]
-paths = [
-  '''go\.mod''',
-  '''go\.sum'''
-]
-# note: (rule) regexTarget defaults to check the _Secret_ in the finding.
-# if regexTarget is not specified then _Secret_ will be used.
-# Acceptable values for regexTarget are "match" and "line"
-regexTarget = "match"
-regexes = [
-  '''process''',
-  '''getenv''',
-]
-# note: stopwords targets the extracted secret, not the entire regex match
-# like 'regexes' does. (stopwords introduced in 8.8.0)
-stopwords = [
-  '''client''',
-  '''endpoint''',
-]
+# Array of strings used for metadata and reporting purposes.
+tags = ["tag","another tag"]
+
+    # You can include allowlists for a rule to reduce false positives.
+    # A finding will be ignored if any allowlist matches.
+    [[rules.allowlists]]
```

Comment:
@rgmz might be worthwhile to note here with a ⚠️ that this config update was introduced in v8.21.0. Also a note that you can still use `[[rules.allowlist]]`. WDYT?

Reply:
I added a comment. LMK your thoughts.

Reply:
@rgmz I would also add a note that `[rules.allowlist]` still works

Reply:
Done.